eng/versioning/external_dependencies.txt

```diff
@@ -72,6 +72,7 @@ org.eclipse.jetty:jetty-http;9.4.11.v20180605
 org.eclipse.jetty:jetty-server;9.4.11.v20180605
 org.hamcrest:hamcrest-all;1.3
 org.hamcrest:hamcrest-library;2.2
+com.sun.activation:jakarta.activation:1.2.1
```

Comment:
nit: The dependencies are in alphabetical order under their respective headings (regular dependencies, test and maven).

Reply:
Thank you @sima-zhu.